### ISSUE TITLE
how is the text_prototype being used?

### ISSUE THREAD

**Issue by pridy999** [SCXsunchenxi/TEST#3]
In the code, it seems that u didn't calculate the text_prototype alignment loss. As in the initialize of Contrastive_loss, the text_prototype is setting default as False. (self.loss = losses.contrastive_loss.ContrastiveLoss(
            compared_length, nb_random_samples, negative_penalty
        ))
It made me confusing how is the text_prototype being used?

**Comment by EmorZz1G** [comment]
Same question. Have you solved it?

**Comment by changlong-li** [comment]
Same question.
